source/content/tls-compatibility.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Transport Layer Security (TLS) 
+subtitle: How to check that your site supports secure TLS-encrypted connections
+description: Learn how to ensure your site is compatible with the latest security standards.
+contenttype: [guide]
+innav: [false]
+categories: [security]
+cms: [--]
+audience: [development]
+product: [--]
+integration: [--]
+tags: [cms, security]
+contributors: [jazzsequence]
+showtoc: true
+permalink: docs/tls-compatibility
+reviewed: "2025-03-07"
+---
+
+This document describes TLS encryption, when it's used, and how to test TLS compatibility.
+
+## What is TLS?
+
+Transport Layer Security (TLS) is a widely adopted cryptographic security protocol that provides secure communication over a computer network. It is the successor to Secure Sockets Layer (SSL) and is used to secure connections between web servers and clients. One of the primary uses of TLS is to encrypt the communication between web applications and servers, such as when a user logs into a website or submits a form.
+
+A web application uses TLS when communicating with another server or service, such as an API or a payment gateway. As such, it is essential to ensure that your site is compatible with the latest TLS standards to protect your users' data and maintain the security of your site.
+
+A TLS connection is initiated by a "handshake". You can think of this as two computers introducing themselves to each other. As part of the TLS handshake, the two servers will acknowledge each other, determine what cryptographic algorithms to use, exchange session keys and handle authentication. This is when the version of TLS encryption is determined.
+
+## TLS Compatibility
+
+TLS 1.0 and 1.1 were published in 1999 and 2006, respectively. Both were deprecated in 2021 and are considered outdated and insecure. Pantheon servers support TLS 1.2 and 1.3 connections. This means any outgoing HTTP requests being made from the _application level_ must be with servers that also support TLS 1.2 or 1.3.
+
+Pantheon application servers will _reject_ connections to applications that do not support TLS 1.2+. <!--TODO: Update to note *when* we will no longer allow TLS 1.2+ connections? -->
```

Comment:
@scottbuscemi If we're publishing this doc before the change, should we put an (anticipated) date?

Reply:
We don't have a firm date yet for the switchover. In the meantime, we can convey that this will be enforced in late 2025.